### PR TITLE
Update README, bump versions

### PR DESCRIPTION
**Pull request by dylanfrankcom**
Update readme for EVENT_BATCH_TIMEOUT changes in moesifapi-python. 
Bump moesifwsgi version.
Bump moesifapi-python version requirement.

Depends on https://github.com/Moesif/moesifapi-python/pull/23

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 requests==2.30.0
 nose==1.3.7
 isodatetimehandler==1.0.2
-moesifapi>=1.4.9
+moesifapi>=1.5.0
```
Comment (praves77) [inline requirements.txt:4]: please update it to `1.5.1`

File: setup.py [hunk 2]
```diff
@@ -90,7 +90,7 @@
     install_requires=[
         'requests>=2.30.0',
         'isodatetimehandler>=1.0.2',
-        'moesifapi>=1.4.9',
+        'moesifapi>=1.5.0',
```
Comment (praves77) [inline setup.py:93]: 1.5.1

**Comment by praves77** [comment]
@dylanfrankcom 
- please update the api version to 1.5.1
- also apply [this change](https://github.com/Moesif/moesifwsgi/pull/67)
- test and lets us know when done.
thanks.

**Review by praves77 — APPROVED** [review]

**Review by keyur9 — APPROVED** [review]
lgtm

**Pull request merged**